components/more-less/more-less.js

```diff
@@ -267,6 +277,7 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 
 	__init_setBaseHeight() {
 		this.__contentHeight = this.height;
+		this.__maxHeight = 'unset';
```

Comment:
I think this should be set to `this.height` since the component is initialized in "shrunken" form.

Reply:
Change has been made, thanks!